src/internal/components/Toast.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useMemo } from 'react';
+import { background, cn } from '../../styles/theme';
+import { closeSvg } from '../svg/closeSvg';
+
+type ToastProps = {
+  className?: string;
+  durationMs?: number;
+  position: 'top-center' | 'top-right' | 'bottom-center' | 'bottom-right';
+  isToastVisible: boolean;
+  setIsToastVisible: (isToastVisible: boolean) => void;
+  children: React.ReactNode;
+};
+
+export function Toast({
+  className,
+  durationMs = 3000,
+  position = 'bottom-center',
+  isToastVisible,
+  setIsToastVisible,
```

Comment:
nit: these could probably just be visible/setIsVisible as they are in the toast

Reply:
happy to change the name, but my intention was to have this state live in the Toast's parent in case developers want to have other triggers close the toast. though i suppose now that we are keeping the component internal, that might not be necessary. 

what do you think, should the state live in this component?

Reply:
ahh, nevermind, I'm sorry I misunderstood your point. 

I will remove the "Toast" from the label of these props in the Toast component, and they'll typically be used something like `isVisible={isToastVisible}`.